pyext/src/output.py

```diff
@@ -801,20 +863,96 @@ def get_fields(self, fields, filtertuple=None, filterout=None, get_every=1):
                     relationship = filtertuple[1]
                     value = filtertuple[2]
                     if relationship == "<":
-                        if float(d[self.invstat2_dict[keytobefiltered]]) >= value:
+                        if float(d[self.inv_dict[keytobefiltered]]) >= value:
                             continue
                     if relationship == ">":
-                        if float(d[self.invstat2_dict[keytobefiltered]]) <= value:
+                        if float(d[self.inv_dict[keytobefiltered]]) <= value:
                             continue
                     if relationship == "==":
-                        if float(d[self.invstat2_dict[keytobefiltered]]) != value:
+                        if float(d[self.inv_dict[keytobefiltered]]) != value:
                             continue
 
-                [outdict[field].append(d[self.invstat2_dict[field]])
+                [outdict[field].append(d[self.inv_dict[field]])
                  for field in fields]
         f.close()
         return outdict
 
+    def return_models_satisfying_criteria(self, criteria):
+        # Given a set of criteria, return lines from the stat file that
```

Comment:
Use " or """ for comments that describe functions (i.e docstrings). Then they'll be picked up and added to the docs.